tests/integration/test_auth.py

```diff
@@ -87,12 +87,12 @@ def test_auth_can_fetch_s3_credentials():
     for daac in earthaccess.daac.DAACS:
         if len(daac["s3-credentials"]) > 0:
             try:
-                print(f"Testing S3 credentials for {daac['short-name']}")
+                logger.info(f"Testing S3 credentials for {daac['short-name']}")
                 credentials = earthaccess.get_s3_credentials(daac["short-name"])
                 assertions.assertIsInstance(credentials, dict)
                 assertions.assertTrue("accessKeyId" in credentials)
             except Exception as e:
-                print(
+                logger.error(
                     f"An error occured while trying to fetch S3 credentials for {daac['short-name']}: {e}"
                 )
```

Comment:
We shouldn't be logging messages from test functions. In fact, I suspect the `try`/`except` should be removed. If an exception occurs during the test, the test should fail. However, I don't know if this is an exception to that rule (no pun intended).

Reply:
https://github.com/botanical/earthaccess/commit/64ec1355f707f1c457415d723f4b43ddaf4ba456 it seems intentional to me according to this commit 😯

Reply:
>In fact, I suspect the try/except should be removed.

>it seems intentional to me according to this commit 😯

It _does_ still seem weird. But let's be intentional about the scope of this PR :) Can we take this to an issue / discussion?

Reply:
> > In fact, I suspect the try/except should be removed.
> 
> > it seems intentional to me according to this commit 😯
> 
> It _does_ still seem weird. But let's be intentional about the scope of this PR :) Can we take this to an issue / discussion?

Sounds good to me. @botanical, would you mind opening a separate issue for this? Also, would you open yet another issue about refactoring loops in tests to instead be parametrized tests (`pytest.parametrize`, and yes, that is how it's spelled, which is apparently a valid variant of _parameterize_).

Reply:
>yes, that is how it's spelled, which is apparently a valid variant of parameterize

Gets me every time

Reply:
Issues added here:

- https://github.com/nsidc/earthaccess/issues/577
- https://github.com/nsidc/earthaccess/issues/578